ios/REANodesManager.m

```diff
@@ -277,7 +320,7 @@ - (void)performOperations
 
       if (canUpdateSynchronously) {
         [strongSelf.uiManager runSyncUIUpdatesWithObserver:strongSelf];
```

Comment:
```suggestion
        [strongSelf.uiManager runSyncUIUpdatesWithObserver:syncUpdateObserver];
```

this change was also in the patch, needed because of the removal of `@interface REANodesManager () <RCTUIManagerObserver>` I think